### PR TITLE
[zh-cn] Resync authentication page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -1137,17 +1124,16 @@ the risks and the mechanisms to protect the CA's usage.
 * `--requestheader-allowed-names` Optional. List of Common Name values (CNs). If set, a valid client certificate with a CN in the specified list must be presented before the request headers are checked for user names. If empty, any CN is allowed.
 -->
 为了防范头部信息侦听，在请求中的头部字段被检视之前，
-身份认证代理需要向 API 服务器提供一份合法的客户端证书，
-供后者使用所给的 CA 来执行验证。
-警告：*不要* 在不同的上下文中复用 CA 证书，除非你清楚这样做的风险是什么以及
-应如何保护 CA 用法的机制。
+身份认证代理需要向 API 服务器提供一份合法的客户端证书，供后者使用所给的 CA 来执行验证。
+警告：*不要* 在不同的上下文中复用 CA 证书，除非你清楚这样做的风险是什么以及应如何保护
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:1128]:
```suggestion
警告：**不要**在不同的上下文中复用 CA 证书，除非你清楚这样做的风险是什么以及应如何保护
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -1460,8 +1457,8 @@ To authenticate against the API:
 * 用户发出 `kubectl` 命令。
 * 凭据插件提示用户输入 LDAP 凭据，并与外部服务交互，获得令牌。
 * 凭据插件将令牌返回该 client-go，后者将其用作持有者令牌提交给 API 服务器。
-* API 服务器使用[Webhook 令牌身份认证组件](#webhook-token-authentication)向
-  外部服务发出 `TokenReview` 请求。
+* API 服务器使用[Webhook 令牌身份认证组件](#webhook-token-authentication)向外部服务发出
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:1460]:
```suggestion
* API 服务器使用 [Webhook 令牌身份认证组件](#webhook-token-authentication)向外部服务发出
```

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md [hunk 3]
```diff
@@ -702,8 +687,8 @@ Providers that don't return an `id_token` as part of their refresh token respons
 
 ##### 选项一 - OIDC 身份认证组件
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/access-authn-authz/authentication.md:688]:
```suggestion
##### 选项一 - OIDC 身份认证组件   {#option-1-oidc-authenticator}
```

Comment (Sea-n) [reply content/zh-cn/docs/reference/access-authn-authz/authentication.md:688]: And `##### 选项二 - 使用 `--token` 选项   {#option-2-use-the-token-option}` missing anchor too.

Comment (tengqm) [reply content/zh-cn/docs/reference/access-authn-authz/authentication.md:688]: These anchors are for the convenience for reference. If we are sure that no other page may reference a (sub)section, adding anchors can be optional.

**Comment by Sea-n** [comment]
Thanks for the explanation!
/lgtm

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md (hunk 3 above)
Comment (howieyuen) [reply content/zh-cn/docs/reference/access-authn-authz/authentication.md:688]: 这里破折号和汉字“一”连在一起看着有点迷惑，改成冒号含义不变；选项二，也可以这样修改
```suggestion
##### 选项一：OIDC 身份认证组件
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/reference/access-authn-authz/authentication.md (hunk 3 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/access-authn-authz/authentication.md:688]: OK. Good suggestion.

**Pull request merged**